services/interop-proxy/mix.exs

```diff
@@ -5,7 +5,7 @@ defmodule InteropProxy.Mixfile do
     [
       app: :interop_proxy,
       version: "0.1.0",
```

Comment:
You gonna bump the version up to 1.0 someday?

Reply:
I mean, technically a release for v0.1 hasn't been cut yet, and typically in releases you start with your v0.x releases until the API is guaranteed to be stable. There's a bunch of little things with the status codes that need fixing first, and certainly can't version before a competition in case they make changes.